src/app/globals.css

```
@@ -1,3 +1,19 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+.gradient {
+    @apply bg-clip-text bg-gradient-to-r from-[#00bfa6] from-35% via-[#00bfa6bd] via-65% to-[#00bfa6a5] to-100% text-transparent;
+}
+
+.bg-page {
+    @apply dark:bg-black bg-white;
```

Even though a pure black/white background is tempting, it can strain the your eyes really fast :frog:

Try something like:
```suggestion
@apply dark:bg-[#0D0D0D] bg-[#FCFCFC] 
```

---

The change is implemented in the project. Thank you, @VCAngel